graphql_client/Cargo.toml

```diff
@@ -1,25 +1,25 @@
 [package]
-name = "graphql_client"
-version = "0.9.0"
 authors = ["Tom Houlé <tom@tomhoule.com>"]
-description = "Typed GraphQL requests and responses"
-repository = "https://github.com/graphql-rust/graphql-client"
-license = "Apache-2.0 OR MIT"
-keywords = ["graphql",  "api", "web", "webassembly", "wasm"]
 categories = ["network-programming", "web-programming", "wasm"]
+description = "Typed GraphQL requests and responses"
 edition = "2018"
+keywords = ["graphql", "api", "web", "webassembly", "wasm"]
+license = "Apache-2.0 OR MIT"
+name = "graphql_client"
+repository = "https://github.com/graphql-rust/graphql-client"
+version = "0.9.0"
 
 [dependencies]
+anyhow = {version = "1.0", optional = true}
 doc-comment = "^0.3"
-anyhow = { version = "1.0", optional = true }
-thiserror = { version = "1.0", optional = true }
-graphql_query_derive = { path = "../graphql_query_derive", version = "0.9.0" }
+graphql_query_derive = {path = "../graphql_query_derive", version = "0.9.0"}
+reqwest = {version = "0.10.7", optional = true}
+serde = {version = "^1.0.78", features = ["derive"]}
 serde_json = "1.0"
-serde = { version = "^1.0.78", features = ["derive"] }
-reqwest = { version = "0.10.4", optional = true }
+thiserror = {version = "1.0", optional = true}
 
-[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
-reqwest = "^0.9"
+# [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+# reqwest = "^0.9"
```

Comment:
I was not sure why this was here in the first place, removing it seemed to resolve some issues I had. But since i did not understand it completely, I left it in here as comments for now